### PR TITLE
Do not use common FJP

### PR DESCRIPTION
**Pull request by honnix**
This is on critical path and trigger is usually very slow due to
datastore operation.

@danielnorberg Let's talk about this.

### CODE REVIEW

File: styx-scheduler-service/src/main/java/com/spotify/styx/TriggerManager.java [hunk 1]
```diff
@@ -68,6 +73,7 @@
     this.time = requireNonNull(time);
     this.storage = requireNonNull(storage);
     this.stats = requireNonNull(stats);
+    this.forkJoinPool = new ForkJoinPool(TRIGGER_CONCURRENCY);
```
Comment (danielnorberg) [inline styx-scheduler-service/src/main/java/com/spotify/styx/TriggerManager.java]: Make the `TriggerManager` `Closeable` and shut down the fjp on scheduler shutdown.

Comment (honnix) [reply styx-scheduler-service/src/main/java/com/spotify/styx/TriggerManager.java]: Good catch, will fix!

Comment (honnix) [reply styx-scheduler-service/src/main/java/com/spotify/styx/TriggerManager.java]: fixed by https://github.com/spotify/styx/pull/449/commits/f4a76013669a7e0123bd1c51698bd1956314f3df

**Comment by danielnorberg** [comment]
👍

**Pull request merged**